pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -111,45 +113,34 @@ def expected_new_customer_spend(self) -> xarray.DataArray:
         # Closed form solution to the posterior of nu
         # Eq 3 from [1], p.3
         mean_spend = p_mean * v_mean / (q_mean - 1)
-        # We could also provide the variance
+        # TODO: We could also provide the variance
```

Comment:
Do you want to create an issue for this TODO? Otherwise, we will forget about it.

Reply:
This is related to https://github.com/pymc-labs/pymc-marketing/issues/760.